modules/googleplay/src/main/scala/cards/nine/googleplay/processes/CardsProcesses.scala

```diff
@@ -126,7 +127,7 @@ class CardsProcesses[F[_]](
     }
 
     for {
-      list ← cacheService.listPending(numApps)
+      list ← toFree(cacheService.listPending(numApps))
       status ← list.traverse[Free[F, ?], (Package, PackageStatus)] { pack ⇒
         for (status ← resolvePendingPackage(pack)) yield (pack, status)
```

Comment:
I am afraid I am a bit confused at first... `pack` is not an `F[A]`. What we need is something like
 `(A => F[B]) => (A => F[(A,B)])`, to tuple the input with the result inside `F`.